.github/workflows/ci.yml

```diff
@@ -22,3 +22,9 @@ jobs:
 
       - name: Build and test frontend
         run: yarn build
+
+      - name: Upload the
```

Comment:
dist folder?

is this artifact actually used? i think we will need `actions/download-artifact@v3` to actually use it?

Reply:
Not really, the only thing you can do for now is download it from the actions page. I thought it's at least someting for now.

I'd push actually using it in the CI to next week.